ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -442,7 +442,6 @@ mac_address_table:
             tags: [ < tag_1 >, < tag_2 > ]
             enabled: < true | false >
             ip_address_virtual: < IPv4_address/Mask >
-            ipv6_address_virtual: < IPv6_address/Mask >
```

Comment:
no replacement?

Reply:
This is just the repeated list item. The same list have a clear description of this and the other var above